### DISCUSSION TITLE
Setting CODE_IMAGE_STR by default

### DISCUSSION THREAD

**Discussion by benzino77** [benzino77/tasmocompiler#167] [Ideas]
I'm wondering if TC should set `CODE_IMG_STR` by default to let's say `TasmoCompiler`?
I just saw this setting in @drhideg answer to #166 and I like this idea.

**Comment by drhideg** [comment]
I'm in. Supplemented with the selected board name/description or part of the name/description would be informative.

**Reply by benzino77** [reply]
Could you give an example of such `CODE_IMAGE_STR`?

**Reply by drhideg** [reply]
I apologize for my clashed writing. In my mind it would be like these:

With name:
 - TasmoCompiler-esp8266
 - TasmoCompiler-esp82664M
 - TasmoCompiler-zbbridge
 - TasmoCompiler-esp32
 - TasmoCompiler-esp32webcam
 - TasmoCompiler-esp32odroid-go
 - TasmoCompiler-esp32m5

With description:
- TasmoCompiler for Generic ESP8266
- TasmoCompiler for Wemos/NodeMCU
- TasmoCompiler for SonOff Zigbee Bridge
- TasmoCompiler for Generic ESP32
- TasmoCompiler for ESP32 webcam
- TasmoCompiler for ESP32 odroid-go
- TasmoCompiler for ESP32 M5Stack Core2

With unique part:
 - TasmoCompiler-8266
 - TasmoCompiler-82664M
 - TasmoCompiler-zb
 - TasmoCompiler-32
 - TasmoCompiler-webcam
 - TasmoCompiler-odroid
 - TasmoCompiler-m5

The first two can be generated from existed AvailableBoards.js.
